### ISSUE TITLE
Improve Lighthouse accessibility score

### ISSUE THREAD

**Issue by stringlytyped** [stringlytyped/cloudnine#29]
Google Lighthouse current gives the playlist page an accessibility score of 74% but we should be aiming for 85% according to our evaluation criteria
